packages/product-features-list/index.js

```diff
@@ -1,20 +1,23 @@
 import Button from '@hashicorp/react-button'
+import classNames from 'classnames'
+import s from './style.module.css'
 
-export default function ProductFeaturesList({ heading, features }) {
+export default function ProductFeaturesList({ heading, features, className }) {
```

Comment:
This was probably one of the most straightforward conversions. Not sure if there's much to comment on in terms of review, but happy to add detail as needed 🙇